Python/bltinmodule.c

```diff
@@ -1109,20 +1111,30 @@ builtin_exec_impl(PyObject *module, PyObject *source, PyObject *globals,
 }
 
 
-/* AC: cannot convert yet, as needs PEP 457 group support in inspect */
+/*[clinic input]
+getattr as builtin_getattr
+
+    v: object
```

Comment:
I feel like we should use the param names used in the old docstring and in the docs (https://docs.python.org/3.10/library/functions.html#getattr): object, name, default. `v` is meaningless. If there's a technical problem with using `object`, let's use `obj` instead.